packages/components/src/DatePicker/DatePicker.css

```diff
@@ -124,13 +124,9 @@
 }
 
 .datePicker
-  :global(
-    .react-datepicker__day--disabled.react-datepicker__day--outside-month
-  ):hover,
+  :global(.react-datepicker__day--disabled.react-datepicker__day--outside-month):hover,
 .datePicker
-  :global(
-    .react-datepicker__day--disabled.react-datepicker__day--outside-month
-  ):focus {
+  :global(.react-datepicker__day--disabled.react-datepicker__day--outside-month):focus {
```

Comment:
These automatically happened in this commit b903b3ff240ae039719a18acc72614a1012aaf58 because I'm upgrading `stylelint-prettier` and `prettier`